x-pack/plugins/monitoring/config.js

```diff
@@ -49,7 +49,8 @@ export const config = (Joi) => {
       enabled: boolean().default(true),
       index: string().default('.monitoring-alerts-6'),
       email_notifications: object({
-        enabled: boolean().default(true)
+        enabled: boolean().default(true),
+        email_address: string().email(),
```

Comment:
We probably want to keep a similar name as the setting getting deprecated. Maybe `default_admin_email`

Reply:
I considered `default_admin_email`, but to me, that implies that the email address can be overridden somewhere else. If I understand it correctly, Monitoring doesn't allow the email address to be changed for cluster alerts.

The `xpack:defaultAdminEmail` advanced setting makes sense as a default option when creating new watcher jobs through the UI, because it really is the default setting, which the user can change.

Reply:
> The xpack:defaultAdminEmail advanced setting makes sense as a default option when creating new watcher jobs through the UI, because it really is the default setting, which the user can change.

Interesting, I didn't realize Watcher UI uses this setting!

I'm OK with how you had it.